digital_land/phase/prune.py

```diff
@@ -30,8 +30,7 @@ class EntityPrunePhase(Phase):
     remove entries with a missing entity
     """
 
-    def __init__(self, issue_log=None, dataset_resource_log=None):
-        self.issues = issue_log
+    def __init__(self, dataset_resource_log=None):
```

Comment:
leave issue_log in for now but raise a depreciation warning otherwise it may break Samriti's API code